validators/tests/pool.ak

```diff
@@ -378,6 +378,7 @@ fn scoop(options: ScoopTestOptions) {
         options.edit_initial_sum_invariant,
         2000000000000000000000,
       ),
+      linear_amplification_manager: None,
```

Comment:
we should add one test exercising this branch, ideally

Reply:
Using this as test bed for a property based test in the next PR, so will be covered by that.